This patch makes telegram-desktop use system-wide qt-plugins, but not static
## Exclusion from first patch
#+	// without this Telegram doesn't start on Ubuntu 17.04 due GTK errors
#+	setenv("QT_STYLE_OVERRIDE", "qwerty", false);
##

diff --git a/Telegram/SourceFiles/main.cpp b/Telegram/SourceFiles/main.cpp
--- a/Telegram/SourceFiles/main.cpp
+++ b/Telegram/SourceFiles/main.cpp
@@ -39,6 +39,8 @@ int main(int argc, char *argv[]) {
 	// both are finished in Application::closeApplication
 	Logs::start(); // must be started before Platform is started
 	Platform::start(); // must be started before QApplication is created
+	QCoreApplication::addLibraryPath("/usr/lib64/qt5/plugins");
+	QCoreApplication::addLibraryPath("/usr/lib/qt5/plugins");
 
 	int result = 0;
 	{
